<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiPlayerClass</class>
 <widget class="QMainWindow" name="MultiPlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
      <item>
       <widget class="QWidget" name="widget_tool" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5,1">
           <property name="spacing">
            <number>3</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>视频目录:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../MultiPlayer.qrc">
               <normaloff>:/tool/resource/points.png</normaloff>:/tool/resource/points.png</iconset>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>原始屏幕</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="radioButton_two">
           <property name="text">
            <string>二分屏</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="radioButton_four">
           <property name="text">
            <string>四分屏</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="radioButton_nine">
           <property name="text">
            <string>九分屏</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>658</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="15,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QOpenGLWidget" name="openGLWidget"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_ctrl" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../MultiPlayer.qrc">
                 <normaloff>:/tool/resource/play-back.png</normaloff>:/tool/resource/play-back.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_play">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../MultiPlayer.qrc">
                 <normaloff>:/tool/resource/play.png</normaloff>:/tool/resource/play.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonStopPlay">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../MultiPlayer.qrc">
                 <normaloff>:/tool/resource/stop.png</normaloff>:/tool/resource/stop.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </i